tests/unit/utils/taskOverDueDiscordMembers.test.ts

```diff
@@ -27,11 +27,11 @@ describe("taskOverDueDiscordMembers()", () => {
       .spyOn(global, "fetch")
       .mockRejectedValueOnce(new Error("Network error"));
 
-    const result = await taskOverDueDiscordMembers();
-
-    expect(result).toEqual(
-      "Oops! We have encountered an internal Server Error"
-    );
+    try {
+      await taskOverDueDiscordMembers();
+    } catch (error: any) {
```

Comment:
can't we log error anywhere

Reply:
> can't we log error anywhere

In discord-slash, only logging we have available is console.log statements

Reply:
The logging is there, only that it will be logged on CF instead of our local in this case